aws/build_package.sh

```diff
@@ -0,0 +1,9 @@
+rm -r lambda-send-email-deploy > /dev/null 2>&1
+mkdir lambda-send-email-deploy
+cp send_email_lambda.py lambda-send-email-deploy/
+cp -r venv/lib/python3.6/site-packages/* lambda-send-email-deploy/
+cd lambda-send-email-deploy/
+zip -r lambda-send-email.zip .
+mv lambda-send-email.zip ../
+cd ..
+rm -r lambda-send-email-deploy/
```

Comment:
Also, you can use this `mktemp -d` to create a temporary folder, with that you can avoid cleaning after you are done, and the name is random, so you do not need to start the script checking if the folder already exists